Decisions/scs-XXXX-v1-requirements-for-container-registry.md

```diff
@@ -0,0 +1,364 @@
+private. Both solutions offer a wide range of features that may or may not attract
+potential users and CSPs.
+
+# Motivation
+
+Our motivation is that there are use cases, where CSP would like to be able to offer
+customers to store and access their container images using the SCS-compliant private
+container registry. The specific use cases should be discussed, but overall the CSPs
+could offer a private container registry as a service or the CSPs could offer a recipe
+for customers to deploy the private registry themselves utilizing the CSP infrastructure.
+In both cases, the private container registry should be SCS-compliant and should fulfill
+a set of needed requirements e.g. for security and privacy.
+
+The idea here and the purpose of this document is to specify what requirements a 
+specific technical container registry implementation (i.e. software solution) needs to 
+fulfill in the context of SCS.
+
+The purpose of this document is also to select an appropriate container registry 
+implementation that meets all defined requirements and makes an architectural 
+decision on what implementation is fully SCS-compliant and will be used by the SCS.
+
+# Design considerations
+
+There are numerous features to look for when evaluating a container registry solution.
+Our decision process goes through two main stages. 
+
+1. [OSS health check](#oss-health-check)
+2. [Required and desirable features check](#required-and-desirable-features-check)
+
+The open-source software (OSS) health check does the first filter stage. This stage is 
+crucial in the context of SCS and container registry implementation should pass it to 
+promote itself to the second consideration stage. The second stage does an overview over
+the feature set of open source container registry implementations and map out 
+requirements (and nice-to-haves) against it to have a well-documented decision.
+
+Note: Keep in mind that at the time of writing this document, we’ve made our best effort 
+ to survey the container registry landscape based on publicly available materials.
+ If you find something outdated (with respect to the time of writing this document) or
+ outright erroneous, please submit a PR or raise an issue, and we’ll fix it right away.
+
+## OSS health check
+
+This section evaluates the health of the open-source projects that were selected from 
+the currently available solutions. SCS-compliant software must fulfill all OSS health 
+checks defined by the [OSS-Health](https://github.com/SovereignCloudStack/standards/blob/main/Design-Docs/OSS-Health.md) 
+document. The main health checks are:
+- Four Opens (code is fully open source, community is open and diverse, development 
+  process is open, design process is open)
+- Maturity
+- Security
+- Activity
+- Lock-in risk assessment
+
+Each selected OSS project is evaluated based on the above checks, and it is classified 
+to one of three categories as follows:
```

Comment:
I think it reads better if we were to replace `is classified to one of the three categories` with `is classified into one of three categories`.